src/software/jetson_nano/services/motor.cpp

```diff
@@ -543,23 +552,19 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
     // TODO (#2719): interleave the angular accelerations in here at some point.
```

Comment:
Could you please remove this comment as #2719 is closed?

Reply:
removed